pkg/kubelet/dockertools/docker.go

```diff
@@ -66,6 +66,12 @@ type DockerInterface interface {
 // DockerID is an ID of docker container. It is a type to make it clear when we're working with docker container Ids
 type DockerID string
 
+type DockerName struct {
```

Comment:
Maybe "KubeletContainerName"? We are building the DockerName which is a string, from our own representation of it. We're also integrating other container providers (e.g.: Rocket), so we may not wanna specifically call it Docker.